### PR TITLE
[Compute Public Sector]Updates for 30.01.153

### PR DESCRIPTION
**Pull request by eric-bischoff**
DISA STIG Release findings for 30.01.153

### CODE REVIEW

**Comment by eric-bischoff** [comment]
@Pubs-MV This is the Maxwell Update 1 (30.01.153) DISA STIG Release findings.

File: compute/public_sector/Release_Findings/v30_01_153/v30_01_153.adoc [hunk 1]
```diff
@@ -0,0 +1,127 @@
+:toc: macro
+== Scan results for 30.01.114
```
Comment (Pubs-MV) [inline compute/public_sector/Release_Findings/v30_01_153/v30_01_153.adoc]: @eric-bischoff should this be 30.01.153?

Comment (eric-bischoff) [reply compute/public_sector/Release_Findings/v30_01_153/v30_01_153.adoc]: Yep, thanks for the fix.

**Review by Pubs-MV — COMMENTED** [review]
updated book.yml

File: compute/public_sector/Release_Findings/v30_01_153/v30_01_153.adoc [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+|Umask 027. Application is a non-interactive container. There is no interactive console session with the container.
+
+|===
+
+==== Vulnerabilities full report
+
+You can find the full vulnerabilities report https://docs.google.com/spreadsheets/d/1jZwm_dMBQ5tr0ilEIdGkbLHnQCdj04CxU7o-VSwizuo/edit?usp=sharing[here].
+
+
+[cols="3,4,4,4,4", options="header"]
```
Comment (Pubs-MV) [inline compute/public_sector/Release_Findings/v30_01_153/v30_01_153.adoc]: @eric-bischoff do we need this table? There is no data, we have only a heading

and do I need to update the link to the new spreadsheet too?

**Review by Pubs-MV — APPROVED** [review]

**Pull request merged**